snakemake_wrapper_utils/snakemake.py

```diff
@@ -1,13 +1,11 @@
 def get_mem(snakemake, out_unit="MiB"):
     """
     Obtain requested memory (from resources) and return in given units.
-    If no memory resources found, return 0.
+    If no memory resources found, return a value equivalent to 205 MiB.
     """
 
     # Store memory in MiB
-    mem_mb = snakemake.resources.get("mem_gb", 0.2) * 1024
-    if not mem_mb:
-        mem_mb = snakemake.resources.get("mem_mb", 205)
+    mem_mb = snakemake.resources.get("mem_gb", snakemake.resources.get("mem_mb", 205))
```

Comment:
I think what is now missing is the conversion from gb to mb.

Reply:
right